src/library/Uno.Material/Controls/BottomNavigationBarItem.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Uno.Material.Entities;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+
+namespace Uno.Material.Controls
+{
+	public partial class BottomNavigationBarItem : ToggleButton
+	{
+		public string Header
```

Comment:
Header, but TitleProperty. I think renaming didn't work out :D 

AppBarButton uses "Label" as property name, along with "Icon", Should we use the same? Label sounds right here.

Reply:
Fixed to Label